### ISSUE TITLE
Disallow old versions of CLN

### ISSUE THREAD

**Issue by ca-ruz** [chrisguida/smaug#65]
Smaug should refuse to start if core lightning is too old.  
```
$ cl --version
v24.05
```

**Comment by ca-ruz** [comment]
Also add a feature to re add all of the transactions without having to do a re-scan.

**Comment by chrisguida** [comment]
Yep, if they do a scan but their CLN is too old, they'll need to delete and re-add all their wallets so bookkeeper can get notified of the events. A feature to re-transmit all wallet events to bookkeeper would be fantastic. This actually might already happen on startup? Need to look into it..
